resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js

```diff
@@ -125,32 +130,26 @@ const randomWeighted = (options, probabilities) => {
 
 const buildSelectors = (element, depth, oldCombinator, selectorLength, maxSelectorLength, isMatching) => {
     // if nonMatching we add a view-<random-index> class selector that is guaranteed to not have targeted children in the todoMVC
-    if ((!isMatching && !depth) || selectorLength >= maxSelectorLength || !element)
+    if (depth < 3 || selectorLength >= maxSelectorLength)
```

Comment:
I prefer `depth < 3` for simplicity but I think 
`!element || element.tagName === "HTML" || element.tagName === "BODY"`
is less prone to errors in depth calculations from `getNextDepth`?

Reply:
I'd prefer if you compute the initial depth relative to the main-ui element, that way depth 0 is `.main-ui` and we don't have selectors involving the `body` tag